app/models/miq_worker.rb

```diff
@@ -336,7 +336,7 @@ def self.close_pg_sockets_inherited_from_parent
     end
   end
 
-  def start
+  def start_runner
```

Comment:
i extracted the fork/process creation to a method so the subclass can override and NOT create a fork/process